server/src/main/java/org/opensearch/search/aggregations/bucket/range/RangeAggregator.java

```diff
@@ -70,6 +70,7 @@
 import java.util.function.Function;
 
 import static org.opensearch.core.xcontent.ConstructingObjectParser.optionalConstructorArg;
+import static org.opensearch.search.aggregations.bucket.filterrewrite.DateHistogramAggregatorBridge.segmentMatchAll;
```

Comment:
Importing from `DateHistogramAggregatorBridge` does not look correct. Can we move `segmentMatchAll` to parent class `AggregatorBridge`?

Reply:
I've made some updates to fix failing dateHistogram cases and segmentMatchAll is no longer used.